src/module/item/physical/document.ts

```diff
@@ -188,6 +188,14 @@ abstract class PhysicalItemPF2e<TParent extends ActorPF2e | null = ActorPF2e | n
         return !!this.container?.system.stowing;
     }
 
+    get actionCostToDraw(): ZeroToThree {
```

Comment:
Creating a helper in physical item to figure out the number of actions needed to draw a thing makes sense to me, but I don't know about this name / signature. @stwlam any thoughts?

Reply:
I had a hard time with the name so I'm more than happy to take a better suggestion.

Worth nothing that while I copied the logic from existing weapon draw, it assumes that the draw value is really only 1 or 2, whereas certain items (e.g. the legacy Gloves of Storing) may actually have a retrieval cost of 0.  Not sure if that's out of scope here.

Reply:
Super duper out of scope. There's also the option of moving it to a physical/helpers function and then calling that, in lieu of a better option.